ci-operator/config/openshift/networking-console-plugin/openshift-networking-console-plugin-main__okd-scos.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  origin_scos-4.16_base-nodejs-rhel9:
+    name: scos-4.16
+    namespace: origin
+    tag: base-nodejs-rhel9
+  origin_scos-4.16_base-stream9:
+    name: scos-4.16
+    namespace: origin
+    tag: base-stream9
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.21-openshift-4.16
+images:
+- build_args:
+  - name: TAGS
+    value: scos
+  dockerfile_path: Dockerfile.art
```

Comment:
is this path correct?

Reply:
https://github.com/openshift/networking-console-plugin/blob/main/Dockerfile.art exists and https://github.com/openshift-eng/ocp-build-data/blob/6e4930bc07c72157c6464f4a305d57cf41d5eb05/images/networking-console-plugin.yml#L3 shows it is the configured Dockerfile name.